db/seeds.rb

```diff
@@ -130,7 +130,7 @@
                         brand: 'Burton',
                         image: File.open('app/assets/images/burton_process.jpg')
                       },
-                      { name: 'BURTON x NEIGHBORHOOD Throwback Snowboard',
+                      { name: 'NEIGHBORHOOD Throwback',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.